ui-bookly/components/feed.tsx

```diff
@@ -1,18 +1,95 @@
 'use client';
-import { Grid, Card } from '@mui/material';
+import { Grid, Card, TextField, Container } from '@mui/material';
 import * as React from 'react';
+import { useEffect, useState } from 'react';
+import { OrganisationCard } from './OrganisationCard';
+import { gql, useQuery } from '@apollo/client';
+
+type organisation = { 
+    id: number,
+    name: string
+}
+
+type organisations = organisation[];
 
 const Feed: () => React.ReactElement = () => {
+
+    const OrganisationCardList = () => {
```

Comment:
This would be better in a separate file. And don't use the organisations that was declare in the function scope above. Pass it down as a prop.

Reply:
I've refactored to remove this altogether.